src/components/message/message-card-list/MessagesVirtualizedList.tsx

```diff
@@ -147,6 +147,11 @@ const MessagesVirtualizedList = (props: Props) => {
 
 	const onWheel: React.WheelEventHandler<'div'> = event => {
 		event.persist();
+
+		if (updateStore.isActive) {
+			updateStore.isActive = false;
```

Comment:
I think we disscused this issue, and i dont remember what we've decided, but it looks unintuitive that it stops only on mouse wheel scroll

Reply:
We've decided that if there is no way to also check regular scroll, then we don't need to implement this feature at all